index.html

```diff
@@ -55,9 +55,9 @@ <h1>Introduction</h1>
 
 			<p>This specification is a module of the <a>Web Publication</a> specification. It will describe the requirements to create an Audiobook profile of a Web Publication.</p>
 
-			<p>An Audiobook is defined as a collection of audio resources grouped together by a reading order, metadata, and resources, all contained in a manifest. This Audiobook can live on the Open Web Platform, or packaged using the <a href="https://w3c.github.io/wpub/index.htmlpackaged-web-publications">Packaged Web Publications</a> specification.</p>
+			<p>An Audiobook is defined as a collection of audio resources grouped together by a reading order, metadata, and resources, all contained in a manifest. This Audiobook can live on the Open Web Platform, or packaged using a specification to be determined by the Working Group.</p>
```

Comment:
A minor word of caution would be to make "to be determined by the working group" an ednote, as writing it into the spec prose has a way of leading to it being forgotten over time.